.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers=false
```

Comment:
When pnpm auto installed the peer dependencies it was difficult to diagnose why pnpm installed multiple copies of webpack because the swc/core package that caused it was an auto installed peer dependency not present in package.json.

Reply:
Might `strict-peer-dependencies=true` ([docs](https://pnpm.io/next/npmrc#strict-peer-dependencies)) be useful here?

Reply:
The current setup emits a warning on install if a peer dependency is not met which I think is good enough.